charts/newrelic-infrastructure/templates/kubelet/_host_network.tpl

```diff
@@ -1,7 +1,9 @@
 {{/* Returns whether the kubelet scraper should run with hostNetwork: true based on the user configuration. */}}
 {{- define "nriKubernetes.kubelet.hostNetwork" -}}
 {{- /* `get` will return "" (empty string) if value is not found, and the value otherwise, so we can type-assert with kindIs */ -}}
-{{- if get .Values.kubelet "hostNetwork" | kindIs "bool" -}}
+{{- if (include "newrelic.common.gkeAutopilot" .) -}}
```

Comment:
(Assuming this is the [approach we want to follow](https://github.com/newrelic/nri-kubernetes/pull/916/files#r1382950552))

I think that privilege pods are not allowed with Autopilot, therefore  you should also take into account that

Reply:
Correct, privilege pods are not allowed.  If we move away from this approach, we would just need to document override privilege config for any deployments/daemonset that are enabled.